frontend/context/BalanceProvider.tsx

```diff
@@ -105,9 +106,10 @@ export const BalanceProvider = ({ children }: PropsWithChildren) => {
 
   const totalStakedOlasBalance = useMemo(() => {
     return stakedBalances
-      .filter((walletBalance) => {
```

Comment:
took me ~10 mins debugging and a lil crying to find out what's the problem here 😅

Reply:
yes :DDD